src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -5597,6 +5590,15 @@ trait Typers extends Adaptations with Tags with TypersTracking with PatternTyper
               val fix = runReporting.codeAction("make reference explicit", tree.pos.focusStart, w.fix, w.msg)
               runReporting.warning(tree.pos, w.msg, cat, context.owner, fix)
             })
+            if (currentRun.isScala3) {
+              tree.getAndRemoveAttachment[VirtualStringContext.type].foreach(_ =>
+                if (sym != definitions.StringContextModule)
```

Comment:
Semantic conflict with https://github.com/scala/scala/pull/8806/files#diff-8fc65249c2fd6224a965694c97b422b473d8acec82d1f5995c0f0c3a0cdff175R5618 🫤

Reply:
oh, the blessed name is `symbol` not `sym`.

Reply:
btw we were spoiled by `TypeRef(pre, sym, args)`. Somewhere it's `qualifier` instead of `qual`. I commented somewhere that suddenly names matter.